gradle/libs.versions.toml

```diff
@@ -118,6 +118,7 @@ jackson-bom = { module = "com.fasterxml.jackson:jackson-bom", version.ref = "jac
 jakartaMail = { module = "com.sun.mail:jakarta.mail", version.ref = "jakartaMail" }
 jerseyCommon = { module = "org.glassfish.jersey.core:jersey-common", version.ref = "jerseyCommon" }
 jgit = { module = "org.eclipse.jgit:org.eclipse.jgit", version.ref = "jgit" }
+jgit-http-apache = { module = "org.eclipse.jgit:org.eclipse.jgit.http.apache", version.ref = "jgit" }
```

Comment:
Just a thought for a follow-up: Instead of introducing this new dependency / Apache HTTP, we might want to create our own custom implementation of the `org.eclipse.jgit.transport.http.HttpConnection` interface based on OkHttp and use that.